FindTown/FindTown/Presentation/FavoriteScene/Favorite/Favorite1ViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  Favorite1ViewController.swift
+//  FindTown
+//
+//  Created by 장선영 on 2023/01/18.
+//
+
+import UIKit
+import FindTownCore
+import FindTownUI
+import RxSwift
+
+class Favorite1ViewController: BaseViewController {
+    
+    // MARK: - Properties
+    
+    var viewModel: Favorite1ViewModel?
+    
+    // MARK: - Views
+    
+    let anonymousView = AnonymousView()
+    let isEmptyView = EmptyView()
+    let favoriteTableView = FavoriteTableView()
+
+    // MARK: - Life Cycle
+    
+    init(viewModel: Favorite1ViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Functions
+    
+    override func setupView() {
+        self.title = "찜"
+        self.view.backgroundColor = FindTownColor.back2.color
+    }
+    
+    override func addView() {
+        super.addView()
+        
+        [anonymousView, isEmptyView, favoriteTableView].forEach {
+            self.view.addSubview($0)
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+        
+        let view = self.view.safeAreaLayoutGuide
+
+        NSLayoutConstraint.activate([
+            anonymousView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            anonymousView.topAnchor.constraint(equalTo: view.topAnchor),
+            anonymousView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            anonymousView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+        ])
+        
+        NSLayoutConstraint.activate([
+            isEmptyView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            isEmptyView.topAnchor.constraint(equalTo: view.topAnchor),
+            isEmptyView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            isEmptyView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+        ])
+        
+        NSLayoutConstraint.activate([
+            favoriteTableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            favoriteTableView.topAnchor.constraint(equalTo: view.topAnchor),
+            favoriteTableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            favoriteTableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+        ])
+    }
+    
+    override func bindViewModel() {
+        
+        // MARK: ViewModel - Input
+        
+        self.anonymousView.signUpButton.rx.tap
+            .bind { [weak self] in
+                self?.viewModel?.input.signUpButtonTrigger.onNext(())
+            }
+            .disposed(by: disposeBag)
```

Comment:
```suggestion
        self.anonymousView.signUpButton.rx.tap
            .throttle(.milliseconds(300), scheduler: MainScheduler.instance)
            .bind { [weak self] in
                self?.viewModel?.input.signUpButtonTrigger.onNext(())
            }
            .disposed(by: disposeBag)
```

저도 이번 프로젝트를 하면서 알게됐는데 버튼 중복 입력 방지에 너무 좋은 거 같더라구요..!
방어적인코드..? 라고 해야하나 (사실 여기서는 딱히 중복 방지를 할 필요는 없어보이긴 하지만 .. 😅)
한번 검색해보시고 여기서는 선영님 편하신대로 하셔도 좋을 거 같아요 ㅎㅎㅎㅎ

Reply:
throttle 연산자 지난번에도 코멘트 주셨던거 기억납니다 ㅎㅎㅎ 버튼을 누르면 바로 화면전환이 되어서 필요가 없을 것 같아서 안넣었는데 혹시 모르니까 넣어놓는 것도 괜찮을 것 같아요 감사합니다!